### PR TITLE
#2212 - Metrics Dynamic Field Activity Server Time Usage

### PR DESCRIPTION
**Pull request by kodinkat**
- fixes: #2212

### CODE REVIEW

**Comment by corsacca** [comment]
@kodinkat do we need to keep both?

**Comment by kodinkat** [comment]
> @kodinkat do we need to keep both?
> image

More so for future-proofing; but we can always re-introduce if needed... I'll remove....

File: dt-metrics/records/date-range-activity.js [hunk 1]
```diff
@@ -190,8 +194,8 @@ function project_activity_during_date_range() {
     url.searchParams.set('record_type', post_type);
     url.searchParams.set('field', field_id);
     url.searchParams.set('value', search_param_value );
-    url.searchParams.set('ts_start', payload.ts_start);
-    url.searchParams.set('ts_end', payload.ts_end);
+    url.searchParams.set('ts_start', ts_start);
```
Comment (corsacca) [inline dt-metrics/records/date-range-activity.js:197]: @kodinkat should this be date_start now?

**Comment by corsacca** [comment]
Thanks Karl!

**Pull request merged**